static_src/components/users_invite.jsx

```diff
@@ -0,0 +1,73 @@
+
+/**
+ * Renders a form that allows org users to invite new users
+ * to cloud.gov
+ */
+
+import React from 'react';
+
+import { Form, FormText } from './form';
+import PanelDocumentation from './panel_documentation.jsx';
+import Action from './action.jsx';
+
+import FormStore from '../stores/form_store';
+
+import userActions from '../actions/user_actions';
+
+import { validateString } from '../util/validators';
+
+const USERS_INVITE_FORM_GUID = 'users-invite-form';
+
+export default class UsersInvite extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+    FormStore.create(USERS_INVITE_FORM_GUID);
+
+    this.validateString = validateString().bind(this);
+    this._onChange = this._onChange.bind(this);
+    this._onValidForm = this._onValidForm.bind(this);
+  }
+
+  _onChange() {
+
+  }
+
+  _onValidForm(errs, values) {
+    userActions.fetchUserInvite(values.email.value);
+  }
+
+  render() {
+    return (
+      <div className="test-users-invite">
+        <h2>User invite</h2>
+        <PanelDocumentation>
+          <p>Organizational Managers can add new users below.</p>
+        </PanelDocumentation>
+        <Form
+          guid={ USERS_INVITE_FORM_GUID }
+          classes={ ['users_invite_form'] }
+          ref="form"
+          onSubmit={ this._onValidForm }
+        >
+          <legend>
+            Invite a new user
+          </legend>
+          <FormText
+            formGuid={ USERS_INVITE_FORM_GUID }
+            classes={ ['test-users_invite_name'] }
+            label="User's email"
+            name="email"
+            validator={ this.validateString }
+          />
+          <Action label="submit" type="submit">Invite new user</Action>
+        </Form>
+      </div>
+    );
+  }
+
+}
+
+UsersInvite.propTypes = { };
```

Comment:
Although not all of our files do this, it helps to have propTypes and defaultProps as variables set up top and then set them equal here. This makes the component a little easier to read because it's propTypes are at the top. So

```
// top of file, below imports or so:
const propTypes = {};
const defaultProps = {};

// bottom of file, below component class

}

UsersInvite.propTypes = propTypes;
UsersInvite.defaultProps = defaultProps;
```

This is again one of the annoying things that I'm not sure we can configure our linter for, and a bunch of our files aren't doing it. It'd be nice to configure the linter to catch it and fix all the old files.